core/src/integration-test/java/com/scalar/db/storage/TestUtils.java

```diff
@@ -136,4 +146,51 @@ public static double nextDouble(Random random) {
         throw new AssertionError();
     }
   }
+
+  /**
+   * Add a suffix to the metadata database/namespace/schema name and the consensus-commit
+   * coordinator table name.
+   *
+   * @param config the original config
+   * @param testName used for the suffix
+   * @return config added the suffix
+   */
+  public static DatabaseConfig addSuffix(DatabaseConfig config, String testName) {
```

Comment:
We need this because multiple integration test runs concurrently after this change, so we need to change the metadata database/namespace/schema name and the consensus-commit coordinator table name per test to avoid conflicts.